demos/online-boutique/README.md

```diff
@@ -0,0 +1,331 @@
+     data_sent......................: 58 kB  1.1 kB/s
+     dropped_iterations.............: 12     0.236582/s
+     http_req_blocked...............: avg=23.8µs   min=5.77µs  med=10.32µs  max=603.29µs p(90)=14.95µs  p(95)=22.39µs 
+     http_req_connecting............: avg=8.96µs   min=0s      med=0s       max=453.57µs p(90)=0s       p(95)=0s      
+     http_req_duration..............: avg=209.97ms min=1.93ms  med=139.34ms max=1.33s    p(90)=483.78ms p(95)=586.19ms
+       { expected_response:true }...: avg=228.55ms min=9.8ms   med=185.42ms max=1.33s    p(90)=485.66ms p(95)=587.06ms
+     http_req_failed................: 9.52%  ✓ 56        ✗ 532 
+     http_req_receiving.............: avg=1.18ms   min=71.1µs  med=184.24µs max=96.73ms  p(90)=415.37µs p(95)=557.4µs 
+     http_req_sending...............: avg=49.68µs  min=25.66µs med=45.07µs  max=262.88µs p(90)=66.87µs  p(95)=78.91µs 
+     http_req_tls_handshaking.......: avg=0s       min=0s      med=0s       max=0s       p(90)=0s       p(95)=0s      
+     http_req_waiting...............: avg=208.73ms min=1.73ms  med=138.9ms  max=1.33s    p(90)=483.57ms p(95)=585.94ms
+     http_reqs......................: 588    11.592525/s
+     iteration_duration.............: avg=296.12ms min=2.17ms  med=139.79ms max=50.72s   p(90)=484.77ms p(95)=586.97ms
+     iterations.....................: 589    11.61224/s
+     vus............................: 1      min=1       max=18
+     vus_max........................: 18     min=6       max=18
+
+ERRO[0052] some thresholds have failed 
+```
+</details>
+
+This results seems to indicate that the frontend service is not recovering from faults in the requests to the product catalog service.
+
+### Fix the issue
+
+After the results of the fault injection test revealed that the frontend service is not recovering from faults in the requests to the product catalog service, a [fix was implemented](https://github.com/grafana/microservices-demo/pull/1). This fix adds automatic retries to the requests to the catalog product service.
+
+Let's try this solution to validate if it effectively solves the problem.
+
+We first will need to update the frontend service's deployment with an [image that implements the fix](https://github.com/grafana/microservices-demo/pkgs/container/microservices-demo%2Fproductcatalogservice).
+
+```shell
+kubectl set image deployment frontend server=ghcr.io/grafana/microservices-demo/frontend:retry-aborted-requests
+```
+
+Output:
+```
+deployment.apps/frontend image updated
+```
+
+Let's run the test again:
+
+```shell
+xk6-disruptor run --env SVC_URL=$SVC_URL --env INJECT_FAULTS=1 scripts/test-front-end.js
+```
+
+<details>
+
+<summary>
+
+In the Output we can see now the `checks` metric shows a success rate of almost `100%`, and the threshold has not failed, confirming the fix works as expected.
+
+```
+✓ checks.........................: 99.82% ✓ 581       ✗ 1
+```
+
+(click for detailed output)
+
+</summary>
+
+```shell
```

Comment:
With this being command output we can get rid of the `shell` syntax highlighting tip.